usr/lib/webapp-manager/webapp-manager.py

```diff
@@ -274,7 +274,28 @@ def on_key_press_event(self, widget, event):
 
     def on_remove_button(self, widget):
         if self.selected_webapp != None:
-            self.manager.delete_webbapp(self.selected_webapp)
+            confrmdlg = Gtk.MessageDialog(message_type=Gtk.MessageType.WARNING)
+            confrmdlg.set_transient_for(self.window)
+            confrmdlg.add_buttons(Gtk.STOCK_CANCEL, Gtk.ResponseType.CANCEL, Gtk.STOCK_DELETE, Gtk.ResponseType.YES)
+            
+            icon = GdkPixbuf.Pixbuf.new_from_file(self.selected_webapp.icon)
```

Comment:
Can someone confirm where it's showing the icon of the webapp in the confirmation dialog title bar?